src/peft/tuners/lora.py

```diff
@@ -65,8 +65,8 @@ class LoraConfig(PeftConfig):
             "For example, ['q', 'v'] or '.*decoder.*(SelfAttention|EncDecAttention).*(q|v)$' "
         },
     )
-    lora_alpha: int = field(default=None, metadata={"help": "Lora alpha"})
-    lora_dropout: float = field(default=None, metadata={"help": "Lora dropout"})
+    lora_alpha: int = field(default=1, metadata={"help": "Lora alpha"})
```

Comment:
```suggestion
    lora_alpha: int = field(default=8, metadata={"help": "Lora alpha"})
```

Reply:
Let's have default value to be equal to `r` so that the scaling weight is 1

Reply:
isn't then better to set `r = 0`? all layers have r==0 and alpha==1 as default (see [Linear](https://github.com/huggingface/peft/pull/390/files#diff-81096a477425943325e7beb88649e8cae486dddc200ba8b069733a295a6c0104R502), [Embedding](https://github.com/huggingface/peft/pull/390/files#diff-81096a477425943325e7beb88649e8cae486dddc200ba8b069733a295a6c0104R590),  [Linear8bitLt](https://github.com/huggingface/peft/pull/390/files#diff-81096a477425943325e7beb88649e8cae486dddc200ba8b069733a295a6c0104R673))  or change the default for the layers.

Reply:
No, `r=8` is a good default and it reduces the burden on the user to specify it. I believe `lora_alpha =8` as default is good so that the scaling weight is 1

Reply:
fine, committed your suggestion. I was just wondering if having different defaults between the config and layers could not be somewhat surprising/overlooked and maybe result in stupid bugs